### ISSUE TITLE
loss function 

### ISSUE THREAD

**Issue by andrew-begain** [zcemycl/TF2DeepFloorplan#1]
@zcemycl hi,thanks for your great job, have you tested the loss function balanced_entropy???if yes ,how about the result ?

**Comment by zcemycl** [comment]
@andrew-begain I did use that losses to train the model. You mean you want to see the training plot of those losses from tensorboard?

**Comment by zcemycl** [comment]
@andrew-begain I updated the readme with the loss graphs, including the total loss, loss of room, and loss of boundary.

**Comment by andrew-begain** [comment]
> @andrew-begain I updated the readme with the loss graphs, including the total loss, loss of room, and loss of boundary.

thanks
